pkgs/tools/security/keysmith/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, qtquickcontrols2
+, kirigami2
+, oathToolkit
+}:
+stdenv.mkDerivation rec {
+
+  pname = "Keysmith";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "KDE";
+    repo = "keysmith";
+    rev = "v${version}";
+    sha256 = "15fzf0bvarivm32zqa5w71mscpxdac64ykiawc5hx6kplz93bsgx";
+  };
+
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+
+  buildInputs = [ oathToolkit kirigami2 qtquickcontrols2 qtbase ];
+
+  meta = {
+    description = "OTP client for Plasma Mobile and Desktop.";
```

Comment:
This shouldn't have a period at the end, as the [contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) state.